Apps/Log/Tests iOS/LogIOSUITests.swift

```diff
@@ -1,10 +1,3 @@
-//
-//  LogIOSUITests.swift
-//  Tests iOS
-//
-//  Created by Daniel Tombor on 20/02/2022.
-//
-
```

Comment:
Swiftformat volt ez?

Reply:
Nope, manual change

Reply:
There is a way to get rid of the creation: https://stackoverflow.com/a/60237853/3677221